packages/st2/in-requirements.txt

```diff
@@ -7,4 +7,5 @@ st2reactor
 st2exporter
 st2debug
 st2tests
+stackstorm-runner-orchestra
```

Comment:
Eventually, when st2 code changes are in place, we should do the same with all the runners.

We can also do it dynamically inside `inject-deps` Make target, similar as we do for ``st2client``.